src/Symfony/Component/Process/Process.php

```diff
@@ -1562,4 +1565,19 @@ private function escapeArgument($argument)
 
         return '"'.str_replace(array('"', '^', '%', '!', "\n"), array('""', '"^^"', '"^%"', '"^!"', '!LF!'), $argument).'"';
     }
+
+    private function replacePlaceholder($commandLine, $toReplace)
+    {
+        if (!preg_match_all('/\{([a-zA-Z0-9]+?)\}/', $commandLine, $matches)) {
```

Comment:
I think this could be replaced by a single call to preg_replace
the pattern is missing the underscore also